Resources/Migrations/deltaMigrations.yml

```diff
@@ -122,3 +122,6 @@ FoodMealSoftTaco: FoodTacoShell
 
 # 2024-08-18
 BoxMagazinePistolSubMachineGunRubber: MagazineBoxPistolRubber
+
+# 2024-08-22 - Frontier Mail - Add more to these when they come up as mapped
+MailPAI: MailNFPAI
```

Comment:
```suggestion
```
No one should have ever mapped a mail-based pAI; their fault if they have.

Also, frontier migrations should be in frontierMigrations.yml

Reply:
Blame Velcro for mapping it as part of Tortuga, once I do that then I'll sign off and commit this